.github/release.yml

```diff
@@ -0,0 +1,32 @@
+changelog:
+  categories:
+    - title: 🚀 Features
+      labels:
+        - feature
+        - enhancement
+    - title: 💥 Improvements
+      labels:
+        - improvement
+    - title: 🐞 Bug fixes
+      labels:
+        - bug
+        - Bug:P0
+        - Bug:P1
+        - Bug:P2
+        - Bug:P3
+    - title: 📚 Documentations
+      labels:
+        - documentation
+    - title: 🌐 Other
+      labels:
+        - '*'
+      exclude:
```

Comment:
Can we remove the exclude label?

Reply:
If we remove the exclude, PRs with any labels will be added to this group. include those Features and Bugs.